launcher/container_runner.go

```diff
@@ -334,11 +336,22 @@ func (r *ContainerRunner) refreshToken(ctx context.Context) (time.Duration, erro
 
 	// Get token expiration.
 	claims := &jwt.RegisteredClaims{}
-	_, _, err = jwt.NewParser().ParseUnverified(string(token), claims)
+	_, parts, err := jwt.NewParser().ParseUnverified(string(token), claims)
```

Comment:
It seems like we should just be passing in a different claims structure ('like [`MapClaims`](https://pkg.go.dev/github.com/golang-jwt/jwt/v4#MapClaims)) rather than manually decoding the second segment ourselves.

Reply:
Done